honeybee/door.py

```diff
@@ -194,6 +195,37 @@ def perimeter(self):
         """Get the perimeter of the door."""
         return self._geometry.perimeter
     
+    def horizontal_orientation(self, north_vector=Vector2D(0, 1)):
+        """Get a number between 0 and 360 for the orientation of the door in degrees.
+
+        0 = North, 90 = East, 180 = South, 270 = West
+
+        Args:
+            north_vector: A ladybug_geometry Vector2D for the north direction.
+                Default is the Y-axis (0, 1).
+        """
+        return math.degrees(
+            north_vector.angle_clockwise(Vector2D(self.normal.x, self.normal.y)))
+
+    def cardinal_direction(self, north_vector=Vector2D(0, 1)):
+        """Get text description for the cardinal direction that the door is pointing.
```

Comment:
I know it is named cardinal but I would go for 8 or 16 orientations if that makes sense. Not sure about the exact use case but that will be more flexible.

Reply:
Ok. I'll make this change for all of the objects that have this method.